src/content/en/updates/2017/09/picture-in-picture.md

```diff
@@ -0,0 +1,108 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Let's discuss about a Web API that would allow websites to create a floating video window over the desktop.
+
+{# wf_updated_on: 2017-09-07 #}
+{# wf_published_on: 2017-09-07 #}
+{# wf_tags: news,media #}
+{# wf_featured_image: /web/updates/images/generic/picture-in-picture.png #}
+{# wf_featured_snippet: Let's discuss about a Web API that would allow websites to create a floating video window over the desktop #}
+{# wf_blink_components: Blink>Media #}
+
+# Picture In Picture (PiP) {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Since April 2017, Chrome for [Android O supports Picture In Picture]. It allows
+users to play a `<video>` element in a pinned window that isn't blocked by other
```

Comment:
Is "pinned" window the right word? Perhaps, we say "overlay", "secondary", "small", or "miniature".

Reply:
Done.